content/blog/high-utilization-in-systems/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Teams under high utilization
+date: "2022-04-015T12:30:00.284Z"
+description: "How teams under high utilization tends to lose cohesion"
+---
+
+In this blog post we will discuss team responsibilities through the lenses of [queueing theory](https://en.wikipedia.org/wiki/Queueing_theory). If you are not familiar with the concept don't worry about it, but you will have to trust some of my claims. The main one being the relationship between cycle time and throughput and how a _system running close to capacity tends to have higher cycle times_. Put another way:
+
+> As utilization goes up, lead time goes up as well. Except that lead time goes up exponentially. 
+
+But if I'm doing more, how can I be moving slower? This seems counterintuitive at first and it's the reason I'm writing this blog post.
+
+Let's define what we mean with "High utilization" and what "cycle time" actually is. Once we have these definitions in place we can try to tie those concepts together and see how they impact our teams.
+
+## High utilization
+
+We can define high utilization as team members spending _most of their time_ tackling tasks directly related to their projects. You can think of these tasks as something that _moves the project forward towards completion_. In most cases this will mean feature delivery, but it also applies to operational teams taking care of areas like infrastructure or security.
+
+Everyone has a queue of tasks, and this queue only grows as the number of tasks increases. I'm sure most of you have been in this position before, where it doesn't matter the amount of work you do, there's always new things in the queue. This seems great from a company or management perspective since it implies things are getting done. Or is it? I will argue that it's the opposite. The higher the utilization the longer teams will take to complete their tasks and deliver value.
+
+Again, I know this seems counterintuitive.
+
+## Cycle time
+
+We will use the common definition of cycle time here, which is:
+
+> The cycle time is the amount of time it takes to complete a specific task from start to finish.
+
+If you have more tasks than you can handle it's a given that something will have to wait in the queue, but why would cycle time increase?
+
+**Prioritization**
+
+Increased cognitive load is a big part of the problem.
```

Comment:
I think you've made a good job here describing how prioritization adds to the individuals' cognitive load.

One other point that I think does fall under cognitive load, that can also be discussed is the fact that even if you do end up prioritizing efficiently, just the sheer fact of having a queue with lots of tasks in it is stressful, reduces the ability to concentrate, and increases the cognitive load.

Reply:
That's a very good point. Just knowing that something needs to happen or can happen increases the cognitive load. On-call is a good example. Nothing might happen, but there's always that "stress" that makes me unable to fully relax.

Reply:
Exactly! On-call is a great example. 🙌🏻 👌🏻

Reply:
I'm calling this "psychological stress" 😄